### DISCUSSION TITLE
Auto Charge & Auto Drop

### DISCUSSION THREAD

**Discussion by Kitadrian** [artisan-roaster-scope/artisan#1678] [Q&A]
My SV is set at a max all the time but every time I do increase or decrease something at first, charge is taking place

Also
When I have a background log, it auto drops, please send help

**Comment by MAKOMO** [comment]
Deactivate autoCHARGE and autoDROP from within the events dialog, menu Config >> Events, first tab.

**Reply by Kitadrian** [reply]
Thank you so much! Big help!
